src/main/java/com/ot/restaurant/seeder/AbstractSeeder.java

```
@@ -5,7 +5,12 @@
 public abstract class AbstractSeeder<E> implements Seeder<E> {
 
   public void seed() {
-    List<E> entities = load();
+    List<E> entities = null;
+    try {
+      entities = load();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
```

Let the seeder user to catch the exception, this will hide the exception

---

done